convert2rhel/pkghandler.py

```diff
@@ -225,9 +225,16 @@ def _enable(self):
             set_releasever=False,
         )
         if ret_code:
-            loggerinst.critical(
+            loggerinst.critical_no_exit(
                 "Failed to install subscription-manager packages. See the above yum output for details."
             )
+            raise exceptions.CriticalError(
```

Comment:
I have a feeling that we should keep using only underscores in the ids instead of mixing dashes and underscores. 

If for some reason people will try to parse our IDs to present that to some UI, then some additional logic might need to be done to avoid the dashes we have in the names.

Another reason would be to keep the pattern across the other ids since they have only underscores everywhere, even for cases where a dash would be applied. 

Let me know what you think/feel about this suggestion.